### ISSUE TITLE
Apply some typography to Web Components projects

### ISSUE THREAD

**Issue by damyanpetev** [IgniteUI/igniteui-webcomponents#295]
## Is your feature request related to a problem? Please describe.
Currently projects are created without any font specified which means the default to Times New Roman on Windows (serif) and that looks kind of jarring against components that use predominantly sans-serif fonts:
![image](https://user-images.githubusercontent.com/3198469/149161788-19a2ee0c-855d-44aa-addc-ced5bddd1d2d.png)

## Describe the solution you'd like
Would be nice if we can apply some typography along with the theme to make the app look and feel consistent. You can check out the Blazor projects from the App Builder for one way that can be done, though it'd also be prudent to consult w/ @simeonoff if there's a better way or if those might soon become available through the package (in which case this can be deferred for that)

**Comment by rkaraivanov** [comment]
@simeonoff 

What is the state of this issue and the accompanying PR?

**Comment by simeonoff** [comment]
I don't think we ever agreed on a solution @damyanpetev? We can start looking into it again.
